### PR TITLE
Refs/heads/agatha's slide master

### PR DESCRIPTION
**Pull request by Mcerrato120**

### CODE REVIEW

File: _posts/0000-01-02-AgathaMc.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title:"Welcome to our second slide!"
+---
+Your text
+Use the left arrow to go back!
```
Comment (AgathaMc) [inline _posts/0000-01-02-AgathaMc.md:6]: More

Comment (AgathaMc) [reply _posts/0000-01-02-AgathaMc.md:6]: my-slide
needs more work?

Comment (AgathaMc) [reply _posts/0000-01-02-AgathaMc.md:6]: add AgathaMc's file

Comment (Mcerrato120) [inline _posts/0000-01-02-AgathaMc.md:6]: add AgathaMc's file

Comment (Mcerrato120) [reply _posts/0000-01-02-AgathaMc.md:6]: Change title to AgathaMc's file

**Review by Mcerrato120 — COMMENTED** [review]
Add AgathaMc's file

**Review by Mcerrato120 — COMMENTED** [review]
add AgathaMc's file

**Pull request closed without merging**